pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -560,6 +562,46 @@ func getNodeSecurityGroupIDForLB(compute *gophercloud.ServiceClient, nodes []*v1
 	return nodeSecurityGroupIDs.List(), nil
 }
 
+// getFloatingNetworkIdForLB returns a floating-network-id for cluster.
+func getFloatingNetworkIdForLB(client *gophercloud.ServiceClient) (string, error) {
+	var FloatingNetworkIds []string
```

Comment:
lowercase the first letter, following usual golang private/public conventions.

Reply:
Done.